src/alto-ui/Form/CheckBox/CheckBox.scss

```diff
@@ -0,0 +1,73 @@
+@import '../../scss/inc';
+
+.CheckBox {
+  margin-bottom: 10px;
+}
+
+.CheckBox__input {
+  opacity: 0;
+  position: absolute;
+  height: 1px;
+  width: 1px;
+  overflow: hidden;
+  clip: rect(1px, 1px, 1px, 1px);   
```

Comment:
Can you use `@extend %visually-hidden` instead ?

Reply:
Done thx